plaidbench.py

```diff
@@ -98,9 +100,20 @@ def main():
     if args.fp16:
         from keras.backend.common import set_floatx
         set_floatx('float16')
+
     batch_size = int(args.batch_size)
-    truncation_size = 64 / batch_size
-    epoch_size = truncation_size * batch_size
+    epochs = args.epochs
+    examples = args.examples
+    epoch_size = examples / epochs
+
+    if epochs > examples:
+    	print('epoch > examples; restting both values to default')
+    if examples % batch_size != 0:
+    	print('examples % batch-size != 0; resetting both values to default')
```

Comment:
raise Value('Batch size must evenly divide examples') (or similar text). Don't want the program to do something people don't expect, especially a benchmark program